src/utilities/new_versions.jl

```diff
@@ -287,7 +292,7 @@ function make_pr_for_new_version(
             @info("Commit was a success")
             api_retry() do
                 @mock git_push(
```

Comment:
```suggestion
                # For the first push, we intentionally push to the HTTPS remote (`origin`),
                # because we want to avoid triggering duplicate CI runs.
                @mock git_push(
```